run.sh

```diff
@@ -15,7 +15,8 @@ while true
 do
 cd $pokebotpath
 python pokecli.py -cf $filename
-read -p "Press any button or wait 20 seconds." -r -s -n1 -t 20
-echo `date`"Pokebot"$*" Stopped." 
+read -p "Press any button or wait 20 seconds to continue.
```

Comment:
See https://github.com/PokemonGoF/PokemonGo-Bot/commit/e03f8341bb536dea3bc9753d705ea21e20a31418
Someone need it. It's same as /n in echo.